lib/core/layout/components/header/header.component.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+    selector: 'adf-layout-header',
+    templateUrl: './header.component.html',
+    styleUrls: ['./header.component.scss']
+})
+
+export class HeaderLayoutComponent {
+
+    @Input() title: string;
+    @Input() logo: string = '../assets/logo.png';
+    @Input() color: string;
+    @Output() toggled = new EventEmitter<boolean>();
+    expanded = false;
```

Comment:
Why do we need this? The expanded value is never modified. I think instead of "toggled" output parameter, we can name it "clicked" and it can be an EventEmitter<void> since, we are not using this value